lib/chat_app_web/live/home_live.ex

```diff
@@ -37,4 +42,35 @@ defmodule ChatAppWeb.HomeLive do
     </div>
     """
   end
+
+  def handle_event(
+        "create_room",
+        %{
+          "room_name" => room_name,
+          "owner_name" => owner_name,
+          "is_private" => is_private,
+          "password" => password
+        },
+        socket
+      ) do
+    is_private =
+      if is_private == "on" do
+        true
+      else
+        false
+      end
+
+    room = %{
+      "room_name" => room_name,
+      "owner_name" => owner_name,
+      "max_participants" => 20,
```

Comment:
Is there a particular reason why the room has a capacity of 20 participants? Perhaps that number should be chosen in the modal, although I think it is ok for now to just have it 20 by default.

Reply:
Hardcoded, talked at daily and we said that unless we have something to do with this field (which we will, once we do the handling for joining a room as well), we hardcode it

Reply:
@EndlessRecurrence i agree.
We should be able to set the maximum number of participants as long as it is greater than 2.
But we should have a default value of, let's say, 20.

Reply:
> @EndlessRecurrence i agree.
> We should be able to set the maximum number of participants as long as it is greater than 2.
> But we should have a default value of, let's say, 20.

Alrighty then, will modify